cloudevents-protocol-spec/spec.md

```diff
@@ -130,7 +137,7 @@ The [RabbitMQ `content-type`][content-type] property field is set to the media t
 Example for the [JSON format][json-format]:
 
 ```text
-content-type: application/cloudevents+json; charset=UTF-8
```

Comment:
Did you mean to remove the `charset` ?

The example above and below includes a charset, so I'm not sure whether this is what you intended - it's OK if you did!

Reply:
yep! since is the common denominator on the CE specs, and this backs it up https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types I wanted to add an example that included this

Reply:
Got it, thanks!